src/pot.erl

```diff
@@ -106,7 +106,7 @@ valid_totp(Token, Secret, Opts) ->
                 Token ->
                     true;
                 _ ->
-                    Window = proplists:get_value(window, Opts, 0),
+                    Window = proplists:get_value(addwindow, Opts, 0),
                     case check_candidate(Token, Secret, IntervalsNo - Window, IntervalsNo + Window, Opts) of
```

Comment:
Isn't `addwidow` a number that should multiply the `interval_length`? This is now subtracting/adding the result of `time_interval(Opts)` (big number like 52357080) by a relatively small number, `Window  = addwindow` (-2-1,0,1,2,3).
Also if you really plan to remove support of `window` option, can you update the references of it in the README?

Reply:
`addwindow` is supposed to offset the interval for checking the totp, not change the `interval_length`.